src/main/java/com/synopsys/integration/alert/web/api/audit/AuditEntryActions.java

```diff
@@ -31,102 +31,135 @@
+                ConfigurationJobModel commonConfig = commonDistributionConfig.get();
+                if (commonConfig.isEnabled()) {
+                    distributionEvents = notificationProcessor.processNotifications(commonConfig, List.of(notificationContent));
+                } else {
+                    UUID jobConfigId = commonConfig.getJobId();
+                    logger.warn("The Distribution Job with Id {} was disabled. This notification could not be sent", jobConfigId);
+                    String message = String.format("The Distribution Job is currently disabled. %s", jobConfigId.toString());
+                    return new ActionResponse<>(HttpStatus.GONE, message);
+                }
             } else {
-                UUID jobConfigId = commonDistributionConfig.getJobId();
-                logger.warn("The Distribution Job with Id {} was disabled. This notification could not be sent", jobConfigId);
-                throw new AlertJobMissingException("The Distribution Job is currently disabled.", jobConfigId);
+                distributionEvents = notificationProcessor.processNotifications(List.of(notificationContent));
+            }
+            if (distributionEvents.isEmpty()) {
+                logger.warn("This notification could not be sent. Make sure you have a Distribution Job configured to handle this notification.");
             }
-        } else {
-            distributionEvents = notificationProcessor.processNotifications(List.of(notificationContent));
-        }
-        if (distributionEvents.isEmpty()) {
-            logger.warn("This notification could not be sent. Make sure you have a Distribution Job configured to handle this notification.");
-        }
 
-        for (DistributionEvent event : distributionEvents) {
-            UUID commonDistributionId = UUID.fromString(event.getConfigId());
-            Long auditId = auditUtility.findMatchingAuditId(notificationContent.getId(), commonDistributionId).orElse(null);
-            Map<Long, Long> notificationIdToAuditId = new HashMap<>();
-            notificationIdToAuditId.put(notificationContent.getId(), auditId);
-            event.setNotificationIdToAuditId(notificationIdToAuditId);
-            eventManager.sendEvent(event);
+            for (DistributionEvent event : distributionEvents) {
+                UUID commonDistributionId = UUID.fromString(event.getConfigId());
+                Long auditId = auditUtility.findMatchingAuditId(notificationContent.getId(), commonDistributionId).orElse(null);
+                Map<Long, Long> notificationIdToAuditId = new HashMap<>();
+                notificationIdToAuditId.put(notificationContent.getId(), auditId);
+                event.setNotificationIdToAuditId(notificationIdToAuditId);
+                eventManager.sendEvent(event);
+            }
+            return get();
+        } catch (IntegrationException e) {
+            return new ActionResponse<>(HttpStatus.BAD_REQUEST, e.getMessage());
```

Comment:
Wouldn't this be an INTERNAL_SERVER_ERROR since we return the GONE response if there are issues with the two input fields? Maybe this try-catch is too big and we need two to handle the BAD_REQUEST case and the INTERNAL_SERVER_ERROR case?